### ISSUE TITLE
ChipsInput UI customizable

### ISSUE THREAD

**Issue by AyushiBShah** [pchmn/MaterialChipsInput#12]
Hi,

First of all thank you for such an extensive and customizable library.

Can we make the chips input ui also customizable as right now there is no way to fix the ui as per app's need.

**Comment by pchmn** [comment]
Hi.
Do you have some examples of what you want to customize ?

**Comment by AyushiBShah** [comment]
![screenshot_1493899068](https://cloud.githubusercontent.com/assets/17173974/25702528/086db8c8-30ef-11e7-9edc-fff4b1ff8e28.png)
If in some way I could modify the height , background ,etc of the chips input . Also provideleft or right drawable same as we provide for EditText

**Comment by pchmn** [comment]
Yeah ok I see. Indeed it could be great features. I will look into it. But I think it will be for the v2 of the library.
